pkg/apis/kops/networking.go

```diff
@@ -434,6 +446,24 @@ type CiliumNetworkingSpec struct {
 	// BPFCTGlobalAnyMax is the maximum number of entries in the non-TCP CT table.
 	// Default: 262144
 	BPFCTGlobalAnyMax int `json:"bpfCTGlobalAnyMax,omitempty"`
+	// BPFLBAcceleration is the load balancing acceleration mode via XDP ("native", "disabled")
```

Comment:
This one isn't available on any supported distros. It require kernel 5.6+.

Reply:
aha.. removed ^^